src/main/scala/org/constellation/rewards/DTM.scala

```diff
@@ -0,0 +1,29 @@
+package org.constellation.rewards
+
+trait DTM {
```

Comment:
You can remove this interface. Nothing relies on it nor implements it.

Reply:
Good catch, I'll use it